Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp

```diff
@@ -619,6 +636,28 @@ static void webkit_web_context_class_init(WebKitWebContextClass* webContextClass
             nullptr,
             static_cast<GParamFlags>(WEBKIT_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY));
 
+    /**
+     * WebKitWebContext:application-id:
+     *
+     * The reverse DNS application ID for this web context. Setting this property is specially
+     * required when the Application creating WebViews is not spawned using #GApplication nor
+     * Flatpak. In this scenario the internal WebKit process launcher spawns its sub-processes in a
+     * Bubblewrap sandbox that requires some form of application identifier.
+     *
+     * If no application ID is provided the WebContext will attempt to use the #GApplication
+     * identifier. If that fails and the WebProcess sandbox remains enabled (which is the default
+     * behavior since 2.40) the UIProcess will crash, unless the user has explicitly opted-out of
+     * sandboxing using the `WEBKIT_DISABLE_SANDBOX_THIS_IS_DANGEROUS=1` environment variable.
```

Comment:
```suggestion
     * identifier. If that fails and the web process sandbox remains enabled, the application will crash.
```

The default behavior didn't change in 2.40. It's always enabled for new API version and disabled for old API version.

Also, please don't document the environment variable. The sandbox is mandatory for applications and if applications start using the environment variable, we'll probably remove it.